### DISCUSSION TITLE
creating a server profile with local storage is problematic

### DISCUSSION THREAD

**Discussion by hbxx** [HewlettPackard/POSH-HPOneView#496] [Q&A] [answered]
new-ovserverprofile using a template that has local storage defined gets an error stating that the storage controller might not be installed.

you get the same message in the UI, however you issue OK again to override.

there is no override from the powershell command.

**Comment by ChrisLynchHPE** [comment, marked as answer]
This isn't an issue with the library, but with the server itself.  If you do need to workaround this, I would suggest the following:

```powershell
$serverprofile = New-OVServerProfile -ServerProfileTemplate $spt -Passthru

# Mimic the "Okay" button in the UI when creating the profile
Send-OVRequest -Uri "/rest/server-profiles?force=all" -method POST -body $serverprofile
```

**Comment by ChrisLynchHPE** [comment]
Also, the `-Confirm` parameter can be used to override and "mimic the OK button" in the UI.  That is if the profile object returns in an error state.
